core/state/dump.go

```diff
@@ -134,17 +137,17 @@ func (self *StateDB) dump(c collector, excludeCode, excludeStorage, excludeMissi
 }
 
 // RawDump returns the entire state an a single large object
-func (self *StateDB) RawDump(excludeCode, excludeStorage, excludeMissingPreimages bool) Dump {
+func (db *StateDB) RawDump(excludeCode, excludeStorage, excludeMissingPreimages bool) Dump {
```

Comment:
In the other file you used `s` for the short name. The checker will complain that there are 2 names.

Reply:
Fixing